### PR TITLE
Add doc for remote debugging in vscode

### PR DESCRIPTION
**Pull request by isabellachen**
Fixes #

Related to #13180 

docker/Readme.md

Add documentation for remote debugging in VSCode.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -458,3 +458,60 @@ Below are instructions for starting a debug session in PhpStorm that will listen
 1. Back in the main configuration window, click 'Apply' then 'Ok'.
 
 1. You can now start a debug session by clicking 'Run -> Debug' in the main menu
+
+#### Remote Debugging with VSCode
+
+You'll need:
+
+- PHP Debug plugin installed in VSCode
```
Comment (brbrr) [inline docker/README.md]: Do you think we add a link to the "correct" plugin? I see 2 "PHP Debug" extensions available, adding a link should help figure out which one to install

 ```suggestion
- [PHP Debug](https://github.com/felixfbecker/vscode-php-debug) plugin installed in VSCode
```

**Review by brbrr — CHANGES_REQUESTED** [review]
Thank you for your contribution! This guide is really helpful. 

I left a few minor comments with some suggestions.

File: docker/README.md [hunk 2]
```diff
@@ -458,3 +458,60 @@ Below are instructions for starting a debug session in PhpStorm that will listen
...
+- Xdebug Helper for Google Chrome 
+- The easiest Xdebug for Mozilla Firefox.
```
[the remark this answers is not in the document]
Comment (isabellachen) [reply docker/README.md]: I've changed the link to point to 'Xdebug Helper' for Mozilla Firefox. It's the same as the one for Chrome, except for Firefox.

**Comment by isabellachen** [comment]
@brbrr Thanks for the suggestions, I've committed them and added the link for the Mozilla extension.

**Comment by isabellachen** [comment]
@brbrr Hmm, I rebased against the latest master, I probably shouldn't have done that -_- for this task...

**Comment by isabellachen** [comment]
@brbrr I think I better make a new branch from master and copy over the text from the readme and make a new PR from there, otherwise there could be trouble...

**Comment by jeherve** [comment]
I took care of the rebase for you (following steps similar to the ones described [here](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request)), and pushed a few minor changes.

File: docker/README.md [hunk 3]
```diff
@@ -458,3 +458,65 @@ Below are instructions for starting a debug session in PhpStorm that will listen
...
 
 1. You can now start a debug session by clicking 'Run -> Debug' in the main menu
+
+#### Remote Debugging with VSCode
+
+You'll need:
+
+- [PHP Debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) plugin installed in VSCode
+- If you use Google Chrome, install the [Xdebug Helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?hl=en) extension.
+- If you use Firefox, install [Xdebug Helper](https://addons.mozilla.org/en-GB/firefox/addon/xdebug-helper-for-firefox/) add-on.
```
Comment (jeherve) [inline docker/README.md]: I updated the add-on here as the old one you had previously recommended does not appear to be available in the Firefox add-on library anymore.

**Review by jeherve — APPROVED** [review]
I think this is useful info, it's going to be very helpful for folks using VS Code, thank you. I've made a few minor changes, but overall I think this is good to go.

@brbrr What do you think? Wanna merge this?

**Review by brbrr — APPROVED** [review]

**Pull request merged**